src/item.cpp

```diff
@@ -1523,6 +1523,11 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
                 }
             }
         }
+        
+        if( is_gun() && has_flag( "FIRE_TWOHAND" ) ) {
+            info.push_back( iteminfo( "DESCRIPTION",
+                                      _( "* This weapon needs <info>two free hands</info> to fire." ) ) );
```

Comment:
The repeating crossbow uses a pumping action to fire, so it requires two
hands to fire.  The huge crossbow requires two hands to reload, but
otherwise could be fired one handed if you're ridiculously strong.